client/lib/flags/test/index.js

```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
```

Comment:
If I'm interpreting this correctly this adds ~3k gzipped to our `vendors~build` chunk.

Reply:
Hmm, annoying. Think it's okay to keep for now (until we have SVGs in the Gridicons npm)? The alternative would be to inline the SVG here (which I'd prefer not to, but not feeling super strongly about it).

Reply:
i think i'd rather inline the svg than pull react-dom/server into the client build.

Reply:
it's technically ok here (this is the test) but needs to be cleaned up in `client/lib/flags/index.js`